lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -180,7 +180,7 @@ def copy_image(module, ec2):
 
         module.exit_json(changed=True, image_id=image_id)
     except WaiterError as we:
-        module.fail_json(msg='An error occurred waiting for the image to become available. (%s)' %  we.reason)
+        module.fail_json(msg='An error occurred waiting for the image to become available. (%s)' % str(we))
```

Comment:
Would you mind also importing traceback with the other imports and adding exception=traceback.format_exc() to the end of this fail_json? Example: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#boto3-2